Dockerfile.node-feature-discovery-build

```diff
@@ -1,10 +1,10 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14
+FROM golang:1.20-alpine3.18
 
 ENV GO111MODULE=on
 ENV GOFLAGS=""
 
-RUN yum install -y which
-RUN yum install -y podman docker
+RUN ["apk", "add", "bash", "make", "docker", "curl", "shadow", "git"]
```

Comment:
Same questions as https://github.com/openshift/cluster-nfd-operator/pull/328#discussion_r1308466242

Reply:
answered in [PR 328](https://github.com/openshift/cluster-nfd-operator/pull/328)